src/documentation/pages/components/icon/index.vue

```diff
@@ -0,0 +1,35 @@
+<template lang="pug">
+  div
+    <TitleBar title="Icon" description="Use Icon to render Lumen icons in your project." :tabs="pageTabs" />
+    .chi-grid__container.-pt--3
+      .chi-tabs-panel.-active#examples
+        <Examples />
+      .chi-tabs-panel#properties
+        <Properties />
+      .chi-tabs-panel#accessibility
+        <Accessibility />
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import Examples from './examples/index.vue';
+import PageContentTabs from '../../../components/PageContentTabs.vue';
+import Properties from './_properties.vue';
+import Accessibility from './_accessibility.vue';
+import { standardComponentPageTabs } from '../../../constants/constants';
+
+@Component({
+  components: {
```

Comment:
since this is a web component, please add here:

Vue.config.ignoredElements = ['chi-icon', 'chi-brand'];

check the other web components that are already migrated for reference

Reply:
Is this applicable for all the new components?

Reply:
yes as long as they are web components. All existing components already include this (from PR https://github.com/CenturyLink/Chi/pull/293)